.changeset/floppy-foxes-vanish.md

```diff
@@ -0,0 +1,9 @@
+---
+"@snapwp/blocks": minor
+"@snapwp/query": minor
+"@snapwp/types": minor
+"@snapwp/core": minor
+"@snapwp/next": minor
+---
+
+implement noPropertyAccessFromIndexSignature rule in tsconfig base
```

Comment:
Changesets isnt semver "smart", `minor` bumps x.*y*.z even in prerelease.

Also, conventional commits are still important for our changelog.

```suggestion
"@snapwp/blocks": patch
"@snapwp/query": patch
"@snapwp/types": patch
"@snapwp/core": patch
"@snapwp/next": patch
---

dev: Implement `noPropertyAccessFromIndexSignature` rule in tsconfig base
```